bearlyml23/inc/bearlyml23.h

```diff
@@ -0,0 +1,488 @@
+#define MIE_SSIE_MSK                  (1U << MIE_SSIE_POS)
+#define MIE_VSSIE_POS                 0x02U
+#define MIE_VSSIE_MSK                 (1U << MIE_VSSIE_POS)
+#define MIE_MSIE_POS                  0x03U
+#define MIE_MSIE_MSK                  (1U << MIE_MSIE_POS)
+#define MIE_UTIE_POS                  0x04U
+#define MIE_UTIE_MSK                  (1U << MIE_UTIE_POS)
+#define MIE_STIE_POS                  0x05U
+#define MIE_STIE_MSK                  (1U << MIE_STIE_POS)
+#define MIE_VSTIE_POS                 0x06U
+#define MIE_VSTIE_MSK                 (1U << MIE_VSTIE_POS)
+#define MIE_MTIE_POS                  0x07U
+#define MIE_MTIE_MSK                  (1U << MIE_MTIE_POS)
+#define MIE_UEIE_POS                  0x08U
+#define MIE_UEIE_MSK                  (1U << MIE_UEIE_POS)
+#define MIE_SEIE_POS                  0x09U
+#define MIE_SEIE_MSK                  (1U << MIE_SEIE_POS)
+#define MIE_VSEIE_POS                 0x0AU
+#define MIE_VSEIE_MSK                 (1U << MIE_VSEIE_POS)
+#define MIE_MEIE_POS                  0x0BU
+#define MIE_MEIE_MSK                  (1U << MIE_MEIE_POS)
+#define MIE_SGEIE_POS                 0x0CU
+#define MIE_SGEIE_MSK                 (1U << MIE_SGEIE_POS)
+
+#define MIE_USIP_POS                  0x00U
+#define MIE_USIP_MSK                  (1U << MIE_USIP_POS)
+#define MIP_SSIP_POS                  0x01U
+#define MIP_SSIP_MSK                  (1U << MIP_SSIP_POS)
+#define MIP_VSSIP_POS                 0x02U
+#define MIP_VSSIP_MSK                 (1U << MIP_VSSIP_POS)
+#define MIP_MSIP_POS                  0x03U
+#define MIP_MSIP_MSK                  (1U << MIP_MSIP_POS)
+#define MIE_UTIP_POS                  0x04U
+#define MIE_UTIP_MSK                  (1U << MIE_UTIP_POS)
+#define MIP_STIP_POS                  0x05U
+#define MIP_STIP_MSK                  (1U << MIP_STIP_POS)
+#define MIP_VSTIP_POS                 0x06U
+#define MIP_VSTIP_MSK                 (1U << MIP_VSTIP_POS)
+#define MIP_MTIP_POS                  0x07U
+#define MIP_MTIP_MSK                  (1U << MIP_MTIP_POS)
+#define MIP_SEIP_POS                  0x09U
+#define MIP_SEIP_MSK                  (1U << MIP_SEIP_POS)
+#define MIP_VSEIP_POS                 0x0AU
+#define MIP_VSEIP_MSK                 (1U << MIP_VSEIP_POS)
+#define MIP_MEIP_POS                  0x0BU
+#define MIP_MEIP_MSK                  (1U << MIP_MEIP_POS)
+#define MIP_SGEIP_POS                 0x0CU
+#define MIP_SGEIP_MSK                 (1U << MIP_SGEIP_POS)
+
+
+/* ================================ Peripheral Definitions ================================ */
+
+/* Peripheral Struct Definition */
+typedef struct {
+  __IO uint32_t MSIP0;                          /** MSIP Registers (1 bit wide) */
+  __IO uint32_t MSIP1;                          /** MSIP Registers (1 bit wide) */
+  __IO uint32_t MSIP2;                          /** MSIP Registers (1 bit wide) */
+  __IO uint32_t MSIP3;                          /** MSIP Registers (1 bit wide) */
```

Comment:
I helped you fix this. Bearlyml23 has only 4 cores, so there's only MSIP[0..3] registers